index.html

```diff
@@ -257,14 +252,14 @@ <h2>Contact Me</h2>
                     </p>
                 </div>
                 <div class="contact-me-form flex">
-                    <div class='form flex'>
-                        <p class="form-header flex">Ibrahim</p>
-                        <p class="form-email flex">ibr.s.ahmat@gmail.com</p>
-                        <p class="form-text flex">Write your message here</p>
+                    <form class='form flex' action="https://formspree.io/f/mqknlobw" method="POST">
+                        <input class="form-header flex" placeholder='Ibrahim' type="text" name="fname" maxlength='30' required/>
+                        <input class="form-email flex" placeholder="ibr.5500@gmail.com" type="email" name="email" required/>
+                        <textarea class="form-text flex" placeholder="Write your message here" name="message" maxlength="500" required></textarea>
```

Comment:
- [ ] Kindly delete the fixed width on your form section because it makes your page not take the full width of the screen. Kindly fix it.

![image](https://user-images.githubusercontent.com/35535130/159499139-effbbf94-8ae0-42d7-8c53-5255b608cfb5.png)